test/user.js

```diff
@@ -0,0 +1,95 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  should = chai.should();
+
+describe('users', () => {
+  describe('/users/ POST', () => {
+    it('should create a new user without problems', () => {
```

Comment:
Check way of do tests, return promises like this never fail (because only fail when promise is rejected and you catch the errors). We test async code of [this](https://mochajs.org/#asynchronous-code) way, is a callback `done` called only when all is OK

Reply:
This is not fixed yet